### DISCUSSION TITLE
Enabling and organizing Open Collective's community

### DISCUSSION THREAD

**Discussion by gusaus** [communitycontributorgroup/ccg#2] [Projects]
As mentioned in [another discussion](https://github.com/orgs/communitycontributorgroup/discussions/1), the collective for Community Contributor Group (recently renamed to [FOSS Contributors](https://opencollective.com/fosscontributors)) was initially intended to help [Open Source Collective](https://www.oscollective.org/) support and sustain [open source projects](https://discover.opencollective.com/opensource) they fiscally host. 

Several years before, there was a short-lived community-governed collective called Chapters intended to enable, organize and sustain the entire (online and regional) Open Collective community. While [terminology](https://github.com/opencollective/opencollective/issues/2157) and lack of funds ended up [halting progress](https://github.com/opencollective/opencollective/issues/2157#issuecomment-508248208) and activity for several years, the effort to form the [OSC Working Group](https://docs.google.com/document/d/1PKZwhWuV4-nRHx2BYe5n3ol6cjLo6CmUwWwAnc14Ogo/edit?usp=sharing) became a catalyst for revisiting the larger goals of enabling the entire community. 

After Open Collective accommodated our request to change the channel name to something more appropriate and descriptive, we started to build a new team in [#community-working-group](https://opencollective.slack.com/archives/C94E046SY) (Open Collective Slack).

While it's been slow going without sponsorship or much input from Open Collective, the effort to enable their community has grown organically over the past few weeks. 

In an attempt to keep it going, I've started to parse out info from Slack and [Google docs](https://docs.google.com/document/d/1nZR31Xfa9eWz1-2KQ9PGNfxjJdJLm45YoCSnOw13K8I/edit?usp=sharing) into the following projects: 

- [Open Collective community](https://github.com/orgs/fosscontributors/projects/2) 
- [Open Collective projects](https://github.com/orgs/fosscontributors/projects/1) 

It's still a work in progress, but I'm hoping the project boards will help the community group create structure and organize so it will be easier for Open Collective to value enough to support. 

Would love to hear thoughts from other folks in the community and ecosystem.

**Comment by tekNorah** [comment]
I think Project boards and discussions are perfectly good things to have as long as there is a clear CTA, decision points, or action items.  

This post honestly seems more like an announcement that you did a thing than something you are needing specific feedback on.   Please correct me if I am misreading.

That being said, I will take a look at the boards listed and come back here to provide feedback.

**Reply by gusaus** [reply]
Thanks so much for reviving this discussion, Norah!

The intent of this discussion was to provide some historical context to a group of (mostly new) contributors about how Open Collective's community has evolved since one of the founders invited me to help admin and drive the initial effort to enable and sustain community (and why the initial attempt failed https://github.com/opencollective/opencollective/issues/2157#issuecomment-508248208).

Also wanted to reference the projects I set up and get some feedback about the general purpose and structure along with clarifying anything that we were discussing in Slack at the time...

...or now!!

**Comment by tekNorah** [comment]
### [Open Collective community](https://github.com/orgs/fosscontributors/projects/2)

- Tasks are within scope
- Statuses need to be reviewed and updated

### [Open Collective projects](https://github.com/orgs/fosscontributors/projects/1)\
 - The difference between the views should be more descriptive
 - Altered View 1 to Board type so Is more easily navigated
 - [Support in-kind donations](https://github.com/opencollective/opencollective/issues/245) likely needs to be moved to the Done column since closed
 - Other statuses might need to be updated
 
Overall, the structure seems fine.

@gusaus - were you lookin for any more feedback beyond this?

**Reply by gusaus** [reply]
Unlike most open source projects and communities, most contributions to https://github.com/opencollective are done by a very small [team](https://docs.opencollective.com/help/about/team) of employees and paid contractors (funded by a for-profit entity).

While they initially tried to enable community contributions via the Chapters collective and even a bounty program, it looks like even the latter has been dropped (assuming due to lack of capacity). 

The issues referenced in both projects were reflective of how our team could contribute, we're blocked until the Open Collective team enables the following - 

- [ ] https://github.com/fosscontributors/opencollective/issues/6
- [ ] https://github.com/fosscontributors/opencollective/issues/3

The other blocker has been not having capacity on our end to contribute to ['their' priorities](https://github.com/opencollective/.github/blob/master/profile/README.md) (w/out requiring additional staff/time on their end)...

...until now.

**Reply by gusaus** [reply]
> The issues referenced in both projects were reflective of how our team could contribute, we're blocked until the Open Collective team enables the following -
> 
> -  https://github.com/fosscontributors/opencollective/issues/6
> -  https://github.com/fosscontributors/opencollective/issues/3

Spurred on by some followup by Norah and discussions around helping Open Collective move to to their [new community space](https://opencollective.com/opencollective/updates/new-community-space-discord), I created this issue in the Open Collective community repo on GitHub https://github.com/opencollective/community/issues/19

Will leave a followup comment if/when I hear back from Xavier.
